src/AutoRest.CSharp/Common/Output/Builders/BicepSerializationMethodsBuilder.cs

```diff
@@ -102,29 +138,68 @@ public static SwitchCase BuildBicepReadSwitchCase(SerializableObjectType model,
                 Serializations.BicepFormat,
                 Throw(
                     New.Instance(typeof(InvalidOperationException),
-                    Literal("Bicep deserialization is not supported for this type."))));
+                        Literal("Bicep deserialization is not supported for this type."))));
         }
 
         private static IEnumerable<MethodBodyStatement> WriteSerializeBicep(BicepObjectSerialization objectSerialization)
```

Comment:
can we have this return an array directly?

Reply:
I think it will make the code less readable to use an array here.

Reply:
But I can do a List if you feel strongly.